### PR TITLE
[docs] formatting fix in sdk lander

### PR DESCRIPTION
**Pull request by kbrandwijk**
# Why

# How

# Test Plan

# Checklist

- [X] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [ ] This diff will work correctly for `expo build` (eg: updated `@expo/xdl`).
- [ ] This diff will work correctly for `expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

File: docs/pages/versions/unversioned/index.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ The easiest way to create a bare React Native app with support for the Expo SDK
 Every quarter there is a new Expo SDK release that typically updates to the latest stable version of React Native and includes a variety of bugfixes, features and improvements to the Expo SDK.
 
 | Expo SDK Version | React Native Version |
-| ---------------- | :------------------: |
+| ---------------- | -------------------- |
```
Comment (Simek) [inline docs/pages/versions/unversioned/index.md:36]: `:-:` is the GFM way to center the column content, it doesn't affect docs website display, but works when viewing file on GitHub. 

The redesigned Table component, which is a part of the upcoming docs redesign will support columns align, so I think it won't hurt to just leave this in for now.

File: docs/pages/versions/unversioned/index.md [hunk 2]
```diff
@@ -42,4 +42,4 @@ Every quarter there is a new Expo SDK release that typically updates to the late
 
 ### Support for other React Native versions
 
-Packages in the Expo SDK are intended to support the target React Native version for that SDK. Typically, they will not support older versions of React Native, but they may. When a new version of React Native is released, the latest versions of the Expo SDK packages are typically updated to support it; however, this may take weeks or more, depending on the extent of the changes in the release.
```
Comment (Simek) [inline docs/pages/versions/unversioned/index.md:45]: Also, I have spotted this in few other PRs, it looks like you have the different setting for line breaks control characters or the end of file markers than the others. What OS and editor do you use?

File: docs/pages/versions/unversioned/index.md (hunk 1 above)
Comment (kbrandwijk) [reply docs/pages/versions/unversioned/index.md:36]: Well, the behavior is that it centers that column header only, but not the contents, nor the other column header, so it's just inconsistent...

File: docs/pages/versions/unversioned/index.md (hunk 2 above)
Comment (kbrandwijk) [reply docs/pages/versions/unversioned/index.md:45]: I used the GitHub web editor for these.

File: docs/pages/versions/unversioned/index.md (hunk 1 above)
Comment (kbrandwijk) [reply docs/pages/versions/unversioned/index.md:36]: image

Comment (Simek) [reply docs/pages/versions/unversioned/index.md:36]: I see, let's change this for now. 👍

File: docs/pages/versions/unversioned/index.md (hunk 2 above)
Comment (Simek) [reply docs/pages/versions/unversioned/index.md:45]: Hmm, that's interesting. I will do a bit of the research on the topic, however this is just a comment, not a blocker in any mean. 🙂

**Review by Simek — APPROVED** [review]
LGTM! 🚀

File: docs/pages/versions/unversioned/index.md (hunk 2 above)
Comment (kbrandwijk) [reply docs/pages/versions/unversioned/index.md:45]: I appreciate your attention to detail!

**Pull request merged**